cuda/test/matrix/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 ginkgo_create_test(coo_kernels)
 ginkgo_create_test(csr_kernels)
+ginkgo_create_test(csr_kernels2)
```

Comment:
Why not extend csr_kernels?

Reply:
The csr_kernels has waay too many tests and running them takes a long amount of time. It might make sense to split them across a few files ?